src/digitalmarketplace/components/footer/template.njk

```diff
@@ -0,0 +1,86 @@
+{% from "../../../govuk-frontend/components/footer/macro.njk" import govukFooter -%}
+{{ govukFooter({
+  navigation: [
+    {
+      title: 'Contact',
+      columns: 3,
+      items: [
+        {
+          href: '/help',
+          text: 'Digital Marketplace help'
+        }
+      ]
+    },
+    {
+      title: 'About the Digital Marketplace',
+      items: [
+        {
+          href: 'https://www.gov.uk/guidance/digital-marketplace-buyers-guide',
+          text: 'Services you can buy'
+        },
+        {
+          href: 'https://www.gov.uk/guidance/digital-marketplace-suppliers-guide',
+          text: 'Services you can sell'
+        },
+        {
+          href: 'https://www.gov.uk/government/organisations/crown-commercial-service',
+          text: 'About Crown Commercial Services'
+        },
+        {
+          href: 'https://www.gov.uk/government/organisations/government-digital-service',
+          text: 'About Government Digital Services'
+        },
+        {
+          href: '/g-cloud/suppliers',
```

Comment:
Do we really need to use `url_for('external.suppliers_list_by_prefix')` for the href?

https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/templates/layouts/_footer_categories.html#L30

Reply:
If we need to change the url patterns at any point (say renaming it to `/a-rebranded-framework-name/suppliers`) then it makes things much easier. Is it going to cause problems?

Reply:
It means we have a sort of circular dependency between our apps and the frontend toolkit, and makes testing a little harder.

I couldn't find where these urls are defined in dmutils (the route functions just raise NotImplementedError), are they defined by each app based on the frameworks repo?

Reply:
The `NotImplementedError` is never meant to be called, all those functions are doing is providing the flask route string (eg `/help`), so that `url_for` evaluates to something in the template. The actual following of that path is done by the router (and the PaaS routing functionality) and then the app that the router/PaaS directs the traffic to.

Reply:
Ah yes that makes sense